backends/platform/sdl/win32/win32.cpp

```diff
@@ -166,10 +166,8 @@ Common::String OSystem_Win32::getSystemLanguage() const {
 	char langName[9];
 	char ctryName[9];
 
-	const LCID languageIdentifier = GetUserDefaultUILanguage();
```

Comment:
Why did you just completely rip this call off? If this is not supported in Windows 98, it should be enclosed in a Windows version check, instead of being removed like this

Reply:
Since we're only doing this call to pass the result to `GetLocaleInfo`, we can avoid it entirely by passing `LOCALE_USER_DEFAULT` directly to `GetLocaleInfo`, which works across all Windows versions.
Plus `GetUserDefaultUILanguage` returns a **`LANGID`** while `GetLocaleInfo` takes a **`LCID`,** so the original code looks suspect.

Reply:
You're quite right. We can just use LOCALE_USER_DEFAULT, instead of calling GetUserDefaultUILanguage()